torch_xla/csrc/xla_sharding_util.cpp

```diff
@@ -292,7 +292,8 @@ std::vector<int64_t> ShardingUtil::GetShardShape(
 
 std::vector<std::vector<at::indexing::TensorIndex>>
 ShardingUtil::GetShardIndicesForDevices(
-    const std::vector<int64_t>& shard_shape, const xla::OpSharding sharding,
+    const std::vector<int64_t>& shard_shape,
+    const std::vector<int64_t>& tensor_shape, const xla::OpSharding sharding,
     const std::vector<std::string>& devices) {
   std::vector<std::vector<at::indexing::TensorIndex>> shard_indices(
```

Comment:
It's device by tensor dim, is it? Can we also comment on this - a few words?

Reply:
That's correct, I'll add a comment